x-pack/plugins/timelines/public/components/t_grid/body/index.tsx

```diff
@@ -268,6 +274,7 @@ export const BodyComponent = React.memo<StatefulBodyProps>(
     ]);
 
     const selectedCount = useMemo(() => Object.keys(selectedEventIds).length, [selectedEventIds]);
+    const indexName = useMemo(() => data[0]?._index ?? '', [data]);
```

Comment:
I had to change the `indexName` used in the update component, using `indexNames` prop stopped working.

Reply:
What was the indexNames value before?